_studio/mfx_lib/decode/h265/src/mfx_h265_dec_decode.cpp

```diff
@@ -1084,6 +1084,16 @@ mfxStatus VideoDECODEH265::DecodeFrameCheck(mfxBitstream *bs, mfxFrameSurface1 *
 
         MFXMediaDataAdapter src(bs);
 
+#if (MFX_VERSION >= 1025)
+        mfxExtBuffer* extbuf = (bs) ? GetExtendedBuffer(bs->ExtParam, bs->NumExtParam, MFX_EXTBUFF_DECODE_ERROR_REPORT) : NULL;
+
+        if (extbuf)
+        {
+            ((mfxExtDecodeErrorReport *)extbuf)->ErrorTypes = 0;
```

Comment:
A C++ style `reinterpret_cast` is used below but here the naked C one. Please align.

Reply:
Hi, seems there is also some are using naked C as well.  Is this are required?

Reply:
Add another patch to align the code style.